app/helpers/application_helper.rb

```diff
@@ -120,4 +120,15 @@ def insert_extras(body)
     body
   end
 
+  def render_map(lat, lon, items)
+    
+      var map = L.map('map').setView([lat,lon], 10);
```

Comment:
hmm, so this is JavaScript -- we're going to have to find a way to render a template from a helper, i guess. Needs some research! Perhaps start with searching for "render template from helper in rails"? Or, we could put this in a template like `/app/views/maps/leaflet.js.erb` -- that way it can contain Rails/Ruby code like you've put on line 128-130, see? 

You had an interesting start here, in your comment from #1022:

```ruby
def render_map(lat, lon, items)
  render template : "/app/views/map/_map_helper.html.erb"
end
```

Does that actually work? Like, can we call `render template: ...` in a helper?